src/e2e-test/features/bigquerymultitable/sink/GCSToBQMT_WithMacro.feature

```diff
@@ -0,0 +1,74 @@
+@BQMT_Sink
+Feature: BQMT-Sink - Verify BQMT sink plugin data transfer scenarios
```

Comment:
BQMT-Sink - Verify GCS to BQMT sink plugin data transfer scenarios with macro arguments

Reply:
Change done.